CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 3.0.1 (YYYY-MM-DD)
+
+### Enhancements
```

Comment:
Good idea, then we make sure the order is always the same and we can always delete the empty sections when doing final edits.